core/app/components/forms/elements/RelatedPubsElement.tsx

```diff
@@ -227,40 +238,33 @@ export const RelatedPubsElement = ({
 		return null;
 	}
 
-	const pubsById = useMemo(() => {
-		return pubs.reduce(
-			(acc, pub) => {
-				acc[pub.id] = pub;
-				return acc;
-			},
-			{} as Record<string, ContextEditorPub>
-		);
-	}, [pubs]);
-
-	const linkedPubs = fields.map((f) => f.relatedPubId);
-	const linkablePubs = pubs
-		// do not allow linking to itself or any pubs it is already linked to
-		.filter((p) => p.id !== pubId && !linkedPubs.includes(p.id));
```

Comment:
note: our table's behavior has changed a bit (I think for the better!) where instead of filtering out already linked pubs, we still show them in the table, but they are checked now. however, we don't have an interaction for removing from this table at the moment... i.e. unchecking a box doesn't 'delete' the value right now. I don't think that'd be hard to add though